2023/CharlotteMicrosoftAzure/data.json

```diff
@@ -0,0 +1,29 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "nirajrules",
+      "ashishmgupta"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Charlotte Microsoft Azure",
+    "Country": "United States",
+    "Url": "https://www.meetup.com/charlotte-microsoft-azure1/",
+    "SessionizeApiId": "{Your sessionize event Id}",
```

Comment:
Are you planning to use Sessionize to show off your community's sessions on the global azure site?